### PR TITLE
added countrylookupmethod to v1 docs

### PR DESCRIPTION
**Pull request by olena-microsoft**

### CODE REVIEW

File: api-reference/v1.0/resources/countrynamedlocation.md [hunk 1]
```diff
@@ -55,6 +56,7 @@ The following is a JSON representation of the resource.
 ```json
 {
   "countriesAndRegions": ["String"],
+  "countryLookupMethod": "countryLookedupMethodType",
```
Comment (FaithOmbongi) [inline api-reference/v1.0/resources/countrynamedlocation.md:59]:
```suggestion
  "countryLookupMethod": "String",
```

**Review by FaithOmbongi — COMMENTED** [review]
One thing missing: a changelog object for the new property.

File: api-reference/v1.0/resources/enums.md [hunk 2]
```diff
@@ -11,12 +11,21 @@ doc_type: enumPageType
...
+|authenticatorAppGps|
+
 ### consentRequestFilterByCurrentUserOptions values 
 
 |Member|
 |:---|
 |reviewer|
 |unknownFutureValue|
+
 ### externalEmailOtpState values
```
Comment (davidmu1) [inline api-reference/v1.0/resources/enums.md:30]: Don't forget the changelog: https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/103153/Changelog

Comment (olena-microsoft) [reply api-reference/v1.0/resources/enums.md:30]: Done, thanks.

**Review by FaithOmbongi — APPROVED** [review]

**Comment by olena-microsoft** [comment]
Do not merge until deployment is complete.

**Pull request merged**